aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -955,6 +961,70 @@ void qtopk_kernel(Tensor& values,
   });
 }
 
+template <bool ReluFused>
+void q_batch_norm_kernel(
+    int64_t N,
+    int64_t C,
+    int64_t HxW,
+    const int64_t in_zero_point,
+    const int64_t out_zero_point,
+    const uint8_t* X,
+    const float* alpha,
+    const float* beta,
+    uint8_t* Y) {
+
+  constexpr int kVLen = 8;
+  const int64_t outer_size = N * HxW;
+  using Vec = Vec256<quint8>;
```

Comment:
We can generalize this with AT_FORALL_QINT_TYPES now that we've gotten rid of the bare intrinsics